README.md

```diff
@@ -595,7 +595,7 @@ login.("test")
 
 Always returns ValidResult.
 
-Returns `default_value` in the following cases:
+Returns deeply frozen (see details: [`transform_to_value(value)`](#transform_to_valuevalue)) `default_value` in the following cases:
```

Comment:
Duplicate the text here as the paragraph after "Always returns ValidResult":

```
Returned value is deeply frozen with [Ractor::make_shareable](https://docs.ruby-lang.org/en/master/Ractor.html#method-c-make_shareable) to prevent application bugs due to modification of unintentionally shared value. If that effect is undesired, use [transform { value }](https://github.com/EugZol/datacaster/blob/5df8b17b34f63337852cc436c19b4aae251327c7/README.md#transform--value--) instead.
```

Also replace "freezed" -> "frozen" below.